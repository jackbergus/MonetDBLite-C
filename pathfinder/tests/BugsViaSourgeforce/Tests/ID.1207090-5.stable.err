stderr of test 'ID.1207090-5` in directory 'tests/BugsViaSourgeforce` itself:


# 18:48:26 >  
# 18:48:26 >  Mtimeout -timeout 180 Mserver "--config=/var/tmp/_Stable_PREFIX__GNU_32_d_--enable-debug_/etc/MonetDB.conf" --debug=10 --set "monet_mod_path=/var/tmp/_Stable_PREFIX__GNU_32_d_--enable-debug_/lib/MonetDB:/var/tmp/_Stable_PREFIX__GNU_32_d_--enable-debug_/lib/bin" --set "gdk_dbfarm=/var/tmp/_Stable_PREFIX__GNU_32_d_--enable-debug_/var/MonetDB/dbfarm" --set "sql_logdir=/var/tmp/_Stable_PREFIX__GNU_32_d_--enable-debug_/var/MonetDB/log" --set mapi_port=34046 --set sql_port=43636 --set xquery_port=50796 --set monet_prompt= --trace "--dbname=mTests_tests_BugsViaSourgeforce" --dbinit="module(pathfinder); pfstart();" ; echo ; echo Over..
# 18:48:26 >  


# 18:48:28 >  
# 18:48:28 >  Mtimeout -timeout 60 MapiClient -lxquery -sxml -u monetdb -P monetdb --host="localhost" --port=50796 ID.1207090-5.xq
# 18:48:28 >  


# 18:48:28 >  
# 18:48:28 >  Done.
# 18:48:28 >  

