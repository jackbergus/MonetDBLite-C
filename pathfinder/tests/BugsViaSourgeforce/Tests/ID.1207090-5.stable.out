stdout of test 'ID.1207090-5` in directory 'tests/BugsViaSourgeforce` itself:


# 18:48:26 >  
# 18:48:26 >  Mtimeout -timeout 180 Mserver "--config=/var/tmp/_Stable_PREFIX__GNU_32_d_--enable-debug_/etc/MonetDB.conf" --debug=10 --set "monet_mod_path=/var/tmp/_Stable_PREFIX__GNU_32_d_--enable-debug_/lib/MonetDB:/var/tmp/_Stable_PREFIX__GNU_32_d_--enable-debug_/lib/bin" --set "gdk_dbfarm=/var/tmp/_Stable_PREFIX__GNU_32_d_--enable-debug_/var/MonetDB/dbfarm" --set "sql_logdir=/var/tmp/_Stable_PREFIX__GNU_32_d_--enable-debug_/var/MonetDB/log" --set mapi_port=34046 --set sql_port=43636 --set xquery_port=50796 --set monet_prompt= --trace "--dbname=mTests_tests_BugsViaSourgeforce" --dbinit="module(pathfinder); pfstart();" ; echo ; echo Over..
# 18:48:26 >  

# Monet Database Server V4.8.0_rc1
# Copyright (c) 1993-2005, CWI. All rights reserved.
# Compiled for i686-redhat-linux-gnu/32bit; dynamically linked.
# Visit http://monetdb.cwi.nl/ for further information.

printf("\nReady.\n");

Ready.
quit();

Over..

# 18:48:28 >  
# 18:48:28 >  Mtimeout -timeout 60 MapiClient -lxquery -sxml -u monetdb -P monetdb --host="localhost" --port=50796 ID.1207090-5.xq
# 18:48:28 >  

<?xml version="1.0" encoding="utf-8"?>
<XQueryResult><price>65.95</price></XQueryResult>

# 18:48:28 >  
# 18:48:28 >  Done.
# 18:48:28 >  

